libavformat/movenc.c

```diff
@@ -4430,6 +4433,10 @@ static int mov_write_traf_tag(AVIOContext *pb, MOVMuxContext *mov,
         }
     }
 
+    if (track->cenc.aes_ctr) {
+        ff_mov_cenc_write_stbl_atoms(&track->cenc, pb, moof_offset);
```

Comment:
Does writing these atoms cause any seek inside the stream? Need to integrate with etx and test it.

Reply:
Don't think so. The etx is working here: https://github.com/qluvio/avpipe/commit/838283eb42047b3e61fe4f665fa9d54c8e668c37